editor/limbo_ai_editor_plugin.cpp

```diff
@@ -1893,9 +1895,9 @@ LimboAIEditor::LimboAIEditor() {
 	BASE_CONTROL()->add_child(disk_changed);
 
 	GLOBAL_DEF(PropertyInfo(Variant::STRING, "limbo_ai/behavior_tree/behavior_tree_default_dir", PROPERTY_HINT_DIR), "res://ai/trees");
-	GLOBAL_DEF(PropertyInfo(Variant::STRING, "limbo_ai/behavior_tree/user_task_dir_1", PROPERTY_HINT_DIR), "res://ai/tasks");
-	GLOBAL_DEF(PropertyInfo(Variant::STRING, "limbo_ai/behavior_tree/user_task_dir_2", PROPERTY_HINT_DIR), "");
-	GLOBAL_DEF(PropertyInfo(Variant::STRING, "limbo_ai/behavior_tree/user_task_dir_3", PROPERTY_HINT_DIR), "");
+	PackedStringArray user_task_dir_default;
+	user_task_dir_default.append("res://ai/tasks");
+	GLOBAL_DEF(PropertyInfo(Variant::PACKED_STRING_ARRAY, "limbo_ai/behavior_tree/user_task_dir", PROPERTY_HINT_TYPE_STRING , "4/14:"), user_task_dir_default);
```

Comment:
```suggestion
	GLOBAL_DEF(PropertyInfo(Variant::PACKED_STRING_ARRAY, "limbo_ai/behavior_tree/user_task_dir", PROPERTY_HINT_ARRAY_TYPE, vformat("%s/%s:", Variant::STRING, PROPERTY_HINT_DIR)), user_task_dir_default);
```
This would be better than a magic string; also using `PROPERTY_HINT_ARRAY_TYPE`.